packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -146,8 +148,30 @@ export default class SourceEditing extends Plugin {
 	}
 
 	/**
-	 * Creates source editing wrappers that replace each editing root. Each wrapper contains
-	 *  the document source from the corresponding root.
+	 * @inheritDoc
+	 */
+	afterInit() {
+		const editor = this.editor;
+
+		const pluginNamesToWarn = [
+			'RealTimeCollaborativeEditing',
+			'CommentsEditing',
+			'TrackChangesEditing'
```

Comment:
Let's add the `RevisionHistory` as well :)

Reply:
Done, thanks.